helpers/helpers_emba_dependency_check.sh

```diff
@@ -214,11 +252,17 @@ dependency_check()
       print_output "[*] Info: Proxy settings detected: ${ORANGE}${PROXY_SETTINGS}${NC}" "no_log"
     fi
 
-    if ! curl --connect-timeout 5 -Is https://www.google.com &>/dev/null ; then
+    LATEST_EMBA_VERSION="$(curl --connect-timeout 5 -s -o - https://github.com/e-m-b-a/emba/blob/master/config/VERSION.txt | grep -w "rawLines" | sed -E 's/.*"rawLines":\["([0-9]\.[0-9]\.[0-9]).*/\1/')"
+    if [[ -z "${LATEST_EMBA_VERSION}" ]] ; then
       echo -e "${RED}""not ok""${NC}"
       print_output "[-] Warning: Quest container has no internet connection!" "no_log"
     else
       echo -e "${GREEN}""ok""${NC}"
+      source ./helpers/create_version.sh
+      create_version
```

Comment:
why do we create a version in the dependency checker?

Reply:
where should we create one? Maybe I did not understand the purpose of the script.